math/scip/Portfile

```
@@ -27,6 +27,11 @@ checksums           rmd160  faa34bb47c67cdb7d5789de2e59fc8b09c1cd1f1 \
 
 compilers.setup     require_fortran
 
+# Block compilers that do not support thread-local storage.
+compiler.cxx_standard 2011
+compiler.c_standard   2011
+compiler.thread_local_storage yes
```

The comment doesn't seem necessary since the code is self-explanatory.

Looking at the `-std` flags used during the build, it seems to be using C 1999 and C++ 2014, doesn't it?

I already added `compiler.thread_local_storage yes` below.

```suggestion
compiler.c_standard   1999
compiler.cxx_standard 2014
```